src/components/VDatePicker/mixins/date-picker-table.js

```diff
@@ -85,7 +85,9 @@ export default {
           innerHTML: `<div class="v-btn__content">${this.formatter(value)}</div>`
         },
         on: (this.disabled || !isAllowed) ? {} : {
-          click: () => this.$emit('input', value)
+          click: () => this.$emit('input', value),
+          mouseenter: () => this.$emit('mouseenter', value),
```

Comment:
Please use `mouse-date-enter/leave` names here as well and we'll merge it

Reply:
Done..